README.md

```diff
@@ -159,6 +159,17 @@ acr purge \
     --untagged
 ```
 
+#### Keep flag
+
+To keep at least x number of tags, the `--keep` flag should be set.

```

Comment:
Should we mention the x number is "the latest x number" to make it clear? This applies for other texts.

Reply:
I am not sure if that gives the exact meaning as its applied on top of ago flag ?

For example, if there are 20 tags in a repo that meet the criteria `--ago 3d` , but when we pass `--keep 10` the logic will still leave 20 tags instead of 10. So, it is making sure **at least 10 tags are kept, but not the latest 10 are kept**

Should we rephrase it to say "at least x number of latest tags"

Reply:
The logic gets tags in timedesc order and skips the fore ones, so it actually keeps the most recent ones of the eligible tags. 
I think the idea is just to let users know which tags to keep. For example, suppose there are eligible tags `a20200102`, `b20200103`, `c20200101`, will `--keep 1` keep `a20200102`(first in alphabetical order) or `b20200103`(latest) or `c20200101`(oldest)? Users may be confused...